composer.json

```diff
@@ -4,8 +4,8 @@
   "keywords": ["laravel", "javascript", "language", "translations", "routes"],
   "type": "library",
   "require": {
-    "php": ">=5.5.9",
-    "illuminate/support": "5.1.*|5.2.*|5.3.*|5.4.*|5.5.*"
+    "php": ">=7.1",
```

Comment:
As this is public can we make this change removing 5.5.9 support?

Reply:
I was not aware that it is public. But anyway, I think PHP 5.x is so ancient that if anyone wants to still use the package with such PHP version, will have to stay with 1.0 version. I'll create a new 2.0 tag, so if they have proper semver constraint, they won't be automatically upgraded. Anyway L5.5 requires PHP 7.x, so we have to change PHP constraint here.

Reply:
Sure that is fine